src/ingestion/process-event.ts

```diff
@@ -531,17 +531,21 @@ export class EventsProcessor {
 
         if (await this.shouldSendHooksTask(team)) {
             this.pluginsServer.statsd?.increment(`hooks.send_task`)
-            this.celery.sendTask('ee.tasks.webhooks_ee.post_event_to_webhook_ee', [
-                {
-                    event,
-                    properties,
-                    distinct_id: distinctId,
-                    timestamp,
-                    elements_chain: elementsChain,
-                },
-                team.id,
-                siteUrl,
-            ])
+            this.celery.sendTask(
+                'ee.tasks.webhooks_ee.post_event_to_webhook_ee',
+                [
+                    {
+                        event,
+                        properties,
+                        distinct_id: distinctId,
+                        timestamp,
+                        elements_chain: elementsChain,
+                    },
+                    team.id,
+                    siteUrl,
+                ],
+                {}
```

Comment:
Q: kwargs is currently marked as optional - is it not?

Reply:
hmm, indeed, the `sendTask` function marks them as optional.

However if they're `None`, python/celery fails here

![image](https://user-images.githubusercontent.com/53387/109037413-f91f9500-76ca-11eb-853d-063a92a5f254.png)

I added a `|| {}` to `sendTask` to fix this contract.